lib/Routing/Element/Router.php

```diff
@@ -75,16 +75,17 @@ public function getContext()
      */
     public function supports($name)
```

Comment:
```suggestion
    public function supports(string $name)
```

Reply:
Unfortunately this breaks the code: 
Declaration of Pimcore\Routing\Element\Router::supports(string $name) must be compatible with Symfony\Cmf\Component\Routing\VersatileGeneratorInterface::supports($name)